Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md

```diff
@@ -0,0 +1,183 @@
+﻿---
+title: How to create social media share buttons
+description: Discover how to create share buttons for your Blazor app!
+permalink: /blog/how-to-create-social-media-share-button
+canonical: /blog/how-to-create-social-media-share-button
+image-url: /img/blog/2024-05-17/how-to-create-social-media-share-buttons.png
+image-title: How to create social media share buttons
+author-name: Giorgi
+author-image: giorgi
+posted-on: May 17th, 2024
+read-time: 5 min
+---
+
+# How to create a ShareButton component with [Blazorise](https://blazorise.com/)!
+
+Are you ready to sprinkle some Blazorise magic into your Blazor app?<br/>
+Adding share buttons for social media platforms can give your users an easy<br/>
+way to spread the word about your awesome content. It's easier than you think, thanks to Blazorise!<br/>
+
+---
+
+## Let's dive in and jazz up your app with these fantastic buttons.
+
+## Install Blazorise
+```bash
```

Comment:
For proper formating please add an empty line between blocks and headings.

Reply:
I will do that